### ISSUE TITLE
Support for non latin (Cyrillic) symbols in changelog (Telegram)

### ISSUE THREAD

**Issue by POPSuL** [devnull-tools/build-notifications-plugin#16]
The current message looks like this:

```
Successful - Build #186 of PROJECTNAME

SUCCESS

?????? ????? dist - kisarinnikita

Go to build <https://ci..../186/>
```

In original (git log) this message is "удалил папки dist". Cyrillic author name have the same problem.

**Comment by ataxexe** [comment]
That's odd. I'll investigate it!

Thanks for letting me know.

**Comment by ataxexe** [comment]
I've checked and it's happening with Pushover too. For Boteco is OK (even with Boteco sending messages to Telegram and Pushover).

**Comment by ataxexe** [comment]
I've filed an issue in the Jenkins JIRA since this plugin is now official (see #2).

https://issues.jenkins-ci.org/browse/JENKINS-44220

If you have any other issues, you can open it directly at the Jenkins JIRA instance.

I'll update this issue when the fix is done in the new repository (I'm maintaining this one up for a while, but it will be removed from GitHub).

**Comment by ataxexe** [comment]
It's done! Fixed in the release 1.4.3 (just pushed to the repository, so, please wait a few hours until is available to upgrade).

Again, thank you so much for contributing!

**Comment by POPSuL** [comment]
@ataxexe Nice! Thanks!
